src/server/migrations/0.5.0-0.6.0/migrate.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Recommend removing all of the migration dummy scripts and replacing them with one template script.

Then we will write migrations as needed.